zerver/views/report.py

```diff
@@ -72,7 +79,8 @@ def json_report_error(request, user_profile, message=REQ, stacktrace=REQ,
                       ui_message=REQ(validator=check_bool), user_agent=REQ,
                       href=REQ, log=REQ,
                       more_info=REQ(validator=check_dict([]), default=None)):
-
+    # type: (Any, UserProfile, Any, Any, REQ, Any, Any, Any, REQ) -> HttpResponse
+    # TODO: INCOMPATIBLE_TYPES_WITH_REQ_TYPE_OBJECT_AND_REQ
```

Comment:
Ok,
1. So we should modify it as REQ() from REQ as annotating argument with REQ considers it as REQ() type.
2. Yes, Ideally we should annotate the argument with bool or its known type.. but whenever I tried to do that It gave me the similar error..
`Incompatible types in assignment (expression has type "REQ", variable has type "bool")`